ui/pages/Project/components/FamilyTable/IndividualRow.jsx

```diff
@@ -133,6 +135,21 @@ ShowRnaSeqOutliers.propTypes = {
   sample: PropTypes.object,
 }
 
+const ShowPhenotypePrioritizedGenes = ({ individual, hasPhenotypeGeneScores, ...props }) => (hasPhenotypeGeneScores ? (
+  <Modal
```

Comment:
this is copy-paste from `ShowRnaSeqOutliers`, which means you need a shared base class, or you can define a list of links and do a map, i.e.
```
SHOW_DATA_DETAILS.filter(({ individual, shouldShowField }) => individual[shouldShowField]).map(({ individual, shouldShowField, modalTitle, linkText, elementType, ...props  }) => (
 <Modal
    modalName={toSnakecase(modalTitle)}
    title={modalTitle}
    trigger={<ButtonLink padding="1em 0 0 0" content={linkText} />}
  >
    <React.Suspense fallback={<Loader />}>
      {React.createElement(elementType, props)}
    </React.Suspense>
  </Modal>
))
```

Reply:
Sounds good.